### DISCUSSION TITLE
官方文档打不开

### DISCUSSION THREAD

**Discussion by 2284730142** [umijs/dumi#1930] [Q&A] [answered]
网络正常，无法打开[https://d.umijs.org/](url)

**Comment by PeachScript** [comment, marked as answer]
我这里宽带和 5G 都可以正常打开，dumi 官网是托管在 Vercel 上的，可能你的网络提供方对 Vercel 的 IP 做了拦截，如果有 VPN 可以把 dumi 官网域名加到白名单试下
